globals.json

```diff
@@ -2656,6 +2656,24 @@
 		"uniq": false,
 		"which": false
 	},
+	"vitest": {
```

Comment:
The json file is created by script, please store data in `data/*.js`

Reply:
Got it! [I'll move the changes to data/*.js](https://github.com/sindresorhus/globals/pull/274/files#diff-8f59a9b4760647ce4e81ec0af25a6ac8cccf1588decb56f3404b605a0f070a92) and run the build script instead.

I've updated the PR and force-pushed the changes.